sharding-ui/sharding-ui-frontend/src/components/Container/index.vue

```diff
@@ -18,17 +18,17 @@
 <template>
   <el-container style="height: 100%;">
     <el-aside width="auto">
-      <s-menu :is-collapse="isCollapse"/>
+      <s-menu :is-collapse="isCollapse" />
     </el-aside>
     <el-container>
       <el-header style="padding: 0">
-        <s-head @on-togger="onTogger"/>
+        <s-head class="s-head" @on-togger="onTogger" />
```

Comment:
@wqzwh, maybe we need to improve the configuration of eslint to prevent useless format.

Reply:
add eslint rules，solve the problem of adding self-closing tags，the snippet code is as follows:

```
"vue/html-self-closing": ["error", {
      "html": {
        "void": "any",
        "normal": "any",
        "component": "any"
      },
      "svg": "any",
      "math": "any"
    }]

```